File: Learning_Materials/Beginner_Level/Java/B3IfElseStatement.java
public class B3IfElseStatement {

  public static void main(String[] args) {

    String text; // This is an empty variable, that we will use later

    // If statements are used to check if a condition is true or false
    // They are basically the bread and butter of programming, and are used in
    // almost everything

    // This is an if statement, and it checks if 5 is greater than 2
    if (5 > 2) { 
      text = "value is greater than 2"; // If the condition is true, then this code will run
    }

    // If-Else statements provide an alternative if the condition is false
    if (text == "your mom") {
      System.out.println("Your mom is the best and I would never say anything bad :)");
    } else {
      System.out.println(text); // If the condition is false, then this code will run
    }

    // If-Else-If statements provide multiple alternatives if the condition is false
    if (text == "your mom") {
      System.out.println("your mom is the best and I would never say anything bad :)");
    } else if (text == "value is greater than 2") {
      System.out.println("Your doing good so far! (or are you???)");//If this condition is true, then this code will run
    } else  if (4903 < 0){
      System.out.println("This will never run, because the previous condition is true");
    } else { // you can still add an extra else statement at the end
      System.out.println(text);
    }

  }
}

File: Learning_Materials/Beginner_Level/Python/B3IfElseStatement.py
# If statements are used to check if a condition is true or false
# They are basically the bread and butter of programming, and are used in
# almost everything
text = None

if 5 > 2:  # This is an if statement, and it checks if 5 is greater than 2
    # If the condition is true, then this code will run
    text = "value is greater than 2" 
    # notice how this is indented, as opposed to 
    # other lines we've created. This means that it anything indented
    # below the if statement is "inside" the if statement, and will run


# If-Else statements provide an alternative if the condition is false
if text == "your mom":
    print("Your mom is very nice and I would never say anything bad about her :)")
else:
    print(text)

# If-Else-If statements provide other alternatives if the condition is false
if text == "your mom":
    print("Your mom is very nice and I would never say anything bad about her :)")
elif text == "value is greater than 2":  # elif statement
    print("your doing good so far (or are you???)")
elif 48939018948 < 1:  # you can have more than one elif statement
    print("this will never run")
else:  # You can also still have an else statement
    print(text)
